tests/default_vars.sh

```diff
@@ -42,6 +61,12 @@ elif [[ $MACHINE_ID = wcoss_dell_p3 ]]; then
   TASKS_cpl_bmrk=560; TPN_cpl_bmrk=28; INPES_cpl_bmrk=6; JNPES_cpl_bmrk=8
   THRD_cpl_bmrk=1; WPG_cpl_bmrk=24; MPB_cpl_bmrk="0 287"; APB_cpl_bmrk="0 311"
   OPB_cpl_bmrk="312 431"; IPB_cpl_bmrk="432 479"; WPB_cpl_bmrk="480 559"
+  NPROC_ICE_cpl_bmrk=48
+
+  TASKS_cpl_bmrk_mpi=600; TPN_cpl_bmrk_mpi=40; INPES_cpl_bmrk_mpi=6; JNPES_cpl_bmrk_mpi=8
+  THRD_cpl_bmrk_mpi=1; WPG_cpl_bmrk_mpi=24; MPB_cpl_bmrk_mpi="0 287"; APB_cpl_bmrk_mpi="0 311"
+  OPB_cpl_bmrk_mpi="312 431"; IPB_cpl_bmrk_mpi="432 479"; WPB_cpl_bmrk_mpi="480 599"
+  NPROC_ICE_cpl_bmrk_mpi=48
 
   TASKS_cpl_c192=288; TPN_cpl_c192=28; INPES_cpl_c192=4; JNPES_cpl_c192=8
   THRD_cpl_c192=1; WPG_cpl_c192=12;  MPB_cpl_c192="0 191"; APB_cpl_c192="0 203"
```

Comment:
There are no settings for NPROC_ICE for any coupled c192 test or c384 test. If we're setting this in default_vars, we should be consistent.

I really think that the issue of setting tasking in default_vars is out of control. It think we need to address it, but that is beyond the scope of this PR (Issue [320](https://github.com/ufs-community/ufs-weather-model/issues/320)).

Reply:
I added NPROC_ICE for c192/c384 for consistency.